dimelo/load_processed.py

```diff
@@ -105,31 +143,49 @@ def pileup_vectors_from_bedmethyl(
     motif: str,
     regions: str | Path | list[str | Path],
     window_size: int | None = None,
+    single_strand: bool = False,
+    regions_5to3prime: bool = False,
 ) -> tuple[np.ndarray, np.ndarray]:
     """
-    Generate trace for the specified modification aggregated across all regions in the given bed file. Called by profile plotters, can also be used by a user directly.
-
-    TODO: Merge regions / region_str / window_size handling into a unified function somewhere
-    TODO: How to name this method?
-    TODO: I feel like stuff like this should be shared functionality
-    TODO: I _THINK_ this should return a value for every position, with non-modified positions as zeros
-    TODO: Currently, this redundantly generates a centered bed file and then never uses it; still doing centering in the actual parsing loop.
-        Obvious solution is to just remove the second centering operation and reference the windowed file. But is there a smarter way to structure the code?
+    Generate trace for the specified modification aggregated across all regions in the given bed file.
+    Called by profile plotters, can also be used by a user directly.
+
+    This function loops through all the provided regions and pulls those regions up in your
+    sorted and indexed bedmethyl file. For rows within those regions, we check that the motif
+    is correct (i.e. sequence context, modified base, mod code, and optionally strand). We then add
+    to a pileup vector that spans the size of the first region (by default all regions are assumed to
+    be the same size) and then superimposes all other regions over it. If regions_5to3prime is set to True,
```

Comment:
Nitpick: I think we should avoid having commentary about specific argument settings in the overview section of the doc strings. Haven't thought too hard about it though. Don't change anything just food for thought.